index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Embedding Images and Video</title>
+</head>
+
+<body>
+    <div class="container grid">
+        <header class=" grid">
+            <nav class="navbar">
+                <div class="icon">
+                  <span> </span>
+                  <span> </span>
+                  <span> </span>
+                  </div>
+                <div class="brand">
+                  <img src="https://archive.vn/Bss88/51a3832e6af6792add876216da444db23a4bea96.png" alt="brand">
+                </div>
+                <div class="search">
+                    <div class="box">
+                        <input class="box__search" type="text" placeholder="search">
+                        <a href="#">
+                            <img src="https://image.flaticon.com/icons/svg/483/483356.svg" alt="search"/>
+                        </a>
+                    </div>
+                </div>
+                <div class="tools">
+                    <a href="#"> <img src="https://img.icons8.com/material/24/000000/upload--v1.png" alt="upload" title="Uplaod"> </a>
+                    <a href="#"> <img src="https://img.icons8.com/metro/26/000000/video-call.png" alt="videocall" title="Go live"/> </a>
+                    <a href="#"> <img src="https://img.icons8.com/material-sharp/24/000000/squared-menu.png" alt="menu" title="Apps"/> </a>
+                </div>
```

Comment:
- These 3 embedded images should be beside each other.
![Annotation 2020-06-09 082210](https://user-images.githubusercontent.com/57726348/84100947-8bdbaf00-aa2a-11ea-8685-77b919ea5296.jpg)